libr/bin/p/bin_vsf.c

```diff
@@ -96,6 +104,11 @@ static bool load(RBinFile *bf, RBuffer *b, ut64 loadaddr) {
 			} else if (!CMP_MODULE (VICE_MAINCPU) && module.major == 1) {
 				res->maincpu = R_NEW (struct vsf_maincpu);
 				r_buf_read_at (bf->buf, offset + read, (ut8 *)res->maincpu, sizeof (*res->maincpu));
+			} else {
+				char safe_name[sizeof (module.module_name) + 1];
+				memcpy (safe_name, module.module_name, sizeof (module.module_name));
+				safe_name[sizeof (module.module_name)] = 0;
```

Comment:
uhm maybe you can use r_str_ncpy instead. that function always null terminates

Reply:
oh yeah thanks i always forget that